src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -0,0 +1,80 @@
+package com.google.opensesame.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/user")
+public class UserServlet extends HttpServlet {
+
+  // Dummy list of people @TODO deprecate this
+  private List<PersonObject> people;
+
+  @Override
+  // Instantiate the dummy list of people
+  public void init() {
+    people = new ArrayList<PersonObject>();
+    ArrayList<String> projects = new ArrayList<String>();
+    projects.add("OpenSesame");
+
+    ArrayList<String> ObiSkills = new ArrayList<String>();
+    ObiSkills.add("Meme god");
+    ObiSkills.add("HTML wrangler");
+    PersonObject Obi =
+        new PersonBuilder()
+            .name("Obi")
+            .gitHubID("Obinnabii")
+            .description("Obi is awesome.")
+            .interestTags(ObiSkills)
+            .projectIDs(projects)
+            .buildPerson();
+    people.add(Obi);
+
+    ArrayList<String> SamiSkills = new ArrayList<String>();
+    SamiSkills.add("Stone carver");
+    SamiSkills.add("Bootstrap convert");
+    PersonObject Sami =
+        new PersonBuilder()
+            .name("Sami")
+            .gitHubID("Sami-2000")
+            .description("Sami is fun.")
+            .interestTags(SamiSkills)
+            .projectIDs(projects)
+            .buildPerson();
+    people.add(Sami);
+
+    ArrayList<String> RichiSkills = new ArrayList<String>();
+    RichiSkills.add("Minecraft boss");
+    RichiSkills.add("React wizard");
+    PersonObject Richi =
+        new PersonBuilder()
+            .name("Richi")
+            .gitHubID("Richie78321")
+            .description("Richi is cool.")
+            .interestTags(RichiSkills)
+            .projectIDs(projects)
+            .buildPerson();
+    people.add(Richi);
+  }
+
+  @Override
+  // get a specific user. return null if not found.
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userString = request.getParameter("user");
+
+    PersonObject result =
+        people.stream()
+            .filter(person -> userString.equals(person.getName()))
+            .findAny()
+            .orElse(null);
```

Comment:
Is this intended to query by name? Or is querying by user ID intended here? If the request is querying by name you should definitely be coding for multiple results as there could be multiple users with the same name.

Reply:
It isn't, I was following the mentorServlet convention. Good catch though!